lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -318,6 +318,9 @@ public function createDatabase($database = null)
      */
     public function dropAutoincrement($table)
     {
+        if ($table instanceof Table) {
```

Comment:
@Ocramius please choose the way you want this get patched ... wither this PR or I'll submit another one with the proposed fix as described above - THX